app/models/booking.rb

```diff
@@ -5,4 +5,15 @@ class Booking < ActiveRecord::Base
   	accepts_nested_attributes_for :passengers, reject_if: :all_blank, allow_destroy: true
      # validates :flight_id, presence: true
      # validates :passengers, presence: true
+  def self.paypal_url(paypal_path, id, cost, passenger)
+
+    values = {
+        business: "merchant@ruth-flyapp.com",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.